starter_code/src/com/joelimyx/Driver.java

```diff
@@ -24,12 +24,21 @@ public static void main(String[] args) {
 		
 		//Determine which data structure is fastest for each
 		//operation. Complete the action
-		
+
 		//Delete the first, if any, Student with the last name of "Smith"
-		
+		int x = 0;
+		while (x<19 && !studentArrayList.get(x).getLastName().equalsIgnoreCase("smith")){
+			x++;
+		}
+
+		studentArrayList.remove(x);
```

Comment:
Removal is faster with a linked list than an array list, so if you want the fastest data structure for this operation, it would be linked list